Backpack/BottomSheet/Classes/BackpackFloatingPanelController.swift

```diff
@@ -45,14 +45,12 @@ final class BackpackFloatingPanelController: FloatingPanelController {
     }
 
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        #if swift(>=4.2)
```

Comment:
For some reason this was not being satisfied at runtime 🤷‍♀️
We don't actually need this check anymore though, so 🤪